gu-hub-api.yaml

```
@@ -0,0 +1,560 @@
+swagger: '2.0'
+info:
+  title: Golem unlimited low level hub API
+  description: API description in Markdown.
+  version: 1.0.0
+host: '127.0.0.1:61622'
+tags:
+  - name: hub-session
+    description: HUB Session managment.
+  - name: peer-info
+    description: Unlimited network peer info managment.
+  - name: peer-session
+  - name: lob-man
+    description: Large objects managment
+
+schemes:
+  - http
+security:
+  - gu-api-key: []
+  - systemName: []
+paths:
+  /peer:
```

Rename to be consistent with `/sessions`, but - what's more important - to reflect a plural/array output 
```suggestion
  /peers:
```

---

I was thinking about this.